app/controllers/reservations_controller.rb

```diff
@@ -9,12 +12,14 @@ def new
   end
 
   def create
-    reservation = CreateReservation.new(params[:email], params[:seance_id], params[:seats])
-    if reservation.call
+    new_reservation = Reservations::UseCases::Create.new(user: current_user, seance_id: params[:seance_id],
+                                                         seats: params[:seats])
+
+    if new_reservation.call
       redirect_to seances_path, notice: t('.notice')
     else
       redirect_back fallback_location: new_seance_reservation_path(seance),
-                    alert: reservation.errors.join
+                    alert: new_reservation.errors.join
```

Comment:
I think in current scenario you can have only one in the array. It is ok to join errors 👍 , it wont change anything right now.

Reply:
You are right, thanks:)